src/steps/account.ts

```diff
@@ -5,27 +5,31 @@ import {
 } from '@jupiterone/integration-sdk-core';
 
 import { IntegrationConfig } from '../types';
+import { createAPIClient } from '../client';
 
-export const ACCOUNT_ENTITY_KEY = 'entity:account';
+export const ACCOUNT_ENTITY_KEY = 'at_spoke_account';
```

Comment:
I'm going to update the template project....

We're working toward using this naming pattern for the constant that defines the key for `setData/getData` entries, to clarify that this is meant to be orthogonal to entities and their `_type` values. Then, all things stuck in `setData/getData` will be easily found by `DATA_` search.

```suggestion
export const DATA_ACCOUNT_ENTITY = 'DATA_ACCOUNT_ENTITY';
```

Reply:
Implemented.